pulsar_data_collection/database_connectors/influxdb.py

```diff
@@ -0,0 +1,122 @@
+from typing import Tuple
+
+import pandas as pd
+from influxdb_client import BucketsApi, InfluxDBClient
+from influxdb_client.client.exceptions import InfluxDBError
+from influxdb_client.client.write_api import PointSettings
+
+from .database_actions import DatabaseActions, DatabaseActionsFactory
+
+
+class BatchingCallback(object):
+    def success(self, conf: Tuple[str], data: str):
+        print(f"Written batch: {conf}. \n")
+
+    def error(self, conf: Tuple[str], data: str, exception: InfluxDBError):
+        print(f"Cannot write batch: {conf} \n due: {exception}")
+        raise exception
+
+    def retry(self, conf: Tuple[str], data: str, exception: InfluxDBError):
+        print(f"Retryable error occurs for batch: {conf} \n retry: {exception}")
+
+
+class InfluxdbActions(DatabaseActions):
+    """Infuxdb database connection"""
+
+    def make_connection(self, **kwargs) -> InfluxDBClient:
+        """Makes connection to the Influxdb database
+        Parameters
+        ----------
+        url: str
+        token: str
+        org: str
+        bucket_name: str
+        """
+        if self._check_if_bucket_exists(
```

Comment:
Will this do the same task as expected in your code. 
url=kwargs["url"]
token=kwargs["token"]
org=kwargs["org"], 
bucket_name=kwargs["bucket_name"]
if self._check_if_bucket_exists(url,token,org,bucket_name):
	return InfluxDBClient(url,token,org,bucket_name)

Reply:
yes it will